packages/core/src/common/classes.ts

```diff
@@ -116,8 +116,14 @@ export const CONTROL_GROUP = `${NS}-control-group`;
 export const DIALOG = `${NS}-dialog`;
 export const DIALOG_CONTAINER = `${DIALOG}-container`;
 export const DIALOG_BODY = `${DIALOG}-body`;
+export const DIALOG_BODY_NO_PADDING = `${DIALOG}-body-no-padding`;
+export const DIALOG_BODY_SCROLL_CONTAINER = `${DIALOG}-body-scroll-container`;
 export const DIALOG_CLOSE_BUTTON = `${DIALOG}-close-button`;
 export const DIALOG_FOOTER = `${DIALOG}-footer`;
+export const DIALOG_FOOTER_CONTAINER = `${DIALOG}-footer-container`;
```

Comment:
Needed to apply flex properties on left and right side of the footer. Did not directly added to DIALOG_FOOTER to keep it as current.

Reply:
Hmm, I don't think this is quite working like we want it to. I think we can do this without an additional level of nesting, by using a modifier class instead. I'll take a stab at that approach.

| before | after |
| - | - |
| <img width="583" alt="image" src="https://user-images.githubusercontent.com/723999/211428171-7354d87c-5936-47dd-a58e-45bb234e8ad9.png"> | <img width="583" alt="image" src="https://user-images.githubusercontent.com/723999/211428161-fcbf7df1-79c6-434b-8f9e-8244b48f29f4.png"> |